VERSIONS.md

```diff
@@ -1,5 +1,6 @@
 | Version | iOS version | Android version | Common files version |
 |---------|-------------|-----------------|----------------------|
+| 5.0.0   | 4.7.0       | 5.1.0           | 3.2.1                |
```

Comment:
3.2.2, right? I think we also need to update the Android version number

Reply:
we can do that as a separate thing though since 3.2.3 is already on the way

Reply:
Oh that's right. I'll update this one for now for consistency.